api/v1/views/places.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python3
+"""
+This is module places
+"""
+from api.v1.views import (app_views, Place, storage)
+from flask import (abort, jsonify, make_response, request)
+import os
+
+
+@app_views.route('/cities/<city_id>/places', methods=['GET'],
+                 strict_slashes=False)
+def view_places_in_city(city_id):
+    """
+    all places in a city
+    """
+    city = storage.get("City", city_id)
+    if city is None:
+        abort(404)
+    result = [place.to_json() for place in city.places]
+    return jsonify(result)
+
+
+@app_views.route('/places/<place_id>', methods=['GET'], strict_slashes=False)
+def view_place(place_id=None):
+    """
+    a single place
+    """
+    s = storage.get("Place", place_id)
+    if s is None:
+        abort(404)
+    return jsonify(s.to_json())
+
+
+@app_views.route('/places/<place_id>', methods=['DELETE'],
+                 strict_slashes=False)
+def delete_place(place_id=None):
+    """
+    deleting place
+    """
+    place = storage.get("Place", place_id)
+    if place is None:
+        abort(404)
+    storage.delete(place)
+    return jsonify({}), 200
+
+
+@app_views.route('/cities/<city_id>/places', methods=['POST'],
+                 strict_slashes=False)
+def create_place(city_id):
+    """
+    creates a single place
+    """
+    city = storage.get("City", city_id)
+    if city is None:
+        abort(404)
+    try:
+        r = request.get_json()
```

Comment:
r refers to what?

Reply:
si la ciudad no existe arroja un error estandar 404